controllers/dnschaos/types.go

```diff
@@ -66,9 +71,27 @@ func (r *endpoint) Apply(ctx context.Context, req ctrl.Request, chaos v1alpha1.I
 		return err
 	}
 
-	// TODO: get chaos dns server's address, and send request to this server to set chaos rules.
+	// get dns server's ip used for chaos
+	services, err := utils.SelectAndFilterService(ctx, r.Client, DNSServerSelectorLabels)
```

Comment:
How about using a certain service name instead of label selector for finding coredns instance.

We can setup the service name via `ChaosControllerConfig`.

Reply:
OK, it's a good idea, I will update it

Reply:
done in edc94b6